plugins/PrerenderPlugin/index.js

```diff
@@ -523,6 +539,80 @@ function parsePrerender(html) {
 	return {head, prerender};
 }
 
+// Injects assets into html
```

Comment:
Those 3 functions(injectAssetsIntoHtml, minifyHtml, postProcessHtml) are copies of html-webpack-plugin`s.
Is it impossible that use htmlPluginData.plugin?

Reply:
They are now moved to private functions. I can't reach to the `postProcessHtml` function so that I had to move these to here.